tools/sotn_permuter/permuter_loader.py

```diff
@@ -202,7 +202,8 @@ def sotn_finalize_compile_command(cmdline: importer.List[str]) -> str:
                         [
                             x[-1]
                             for x in segment["subsegments"]
-                            if len(x) > 1
+                            if isinstance(x, list)
```

Comment:
This is a good sanity check even though it shouldn't ever happen.  We all know how well "shouldn't ever happen" holds up.